### PR TITLE
Update and extend NFT token list

### PR DESCRIPTION
**Pull request by masquot**
## Notes for reviewing and merging

The table `nft.tokens` whose entries are updated here is used in `insert` queries that run periodically on:
- '53 * * * *'
- '47 * * * *'

Might be helpful to keep this in mind when merging.

## Data source: [OpenSea](https://opensea.io/)

The data for the NFT contract addresses was pulled directly from the [OpenSea API](https://docs.opensea.io/reference#api-overview)

## Checks 

I've checked that:

* [x] the query produces the intended results
* [x] the folder name matches the schema name
* [x] the schema name exists in Dune
* [x] views are prefixed with `view_`, functions with `fn_`.
* [x] the filename matches the defined view, table or function and ends with .sql
* [x] each file has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`

### CODE REVIEW

**Comment by hildobby** [comment]
LGTM 👍

**Review by hildobby — APPROVED** [review]

File: ethereum/nft/tokens.sql [hunk 1]
```diff
@@ -538,6 +538,4284 @@ COPY nft.tokens (contract_address, name, symbol, standard) FROM stdin;
...
+\\x3e30ca44cd35169bfd98e8ef2c80f06e4c97f6f1	TEMPLATE	TMPLT	ERC721
+\\x41a2788b3599ae8f8fe5da4055e4ad89fdf33ee3	Lite Studios 		ERC721
+\\xa5f4858ada386758e62165037af3fe21655068de	NIFTY HOROSCOPE		ERC721
+\\x5c70591acaf87ddee68b50877d4975e498a32c1f	Unidentified contract		ERC721
+\\x1ae6593db3e5fecb9d1ef5c4ebc81eb46999d6e8	No Labels	NOLABELS	ERC1155
+\\x6caa272b1e9a074a66d82e7f77437e3ba5cdbcd0	Slickmau5 by deadmau5 x OG Slick	SLICKMAUBYDEADMAUXOGSLICK	ERC721
+\\x56b75789580afcbce98db2f16c2cc8d037147da1	The Goose's Gander	by BruceTheGoose	ERC721
+\\x4624573b7a782b3112b23424683f2afe7aed05fb	SUPER PUMP (or BITCOIN - A STRANGE CRITTER)	SUPERPUMP1	ERC721
+\\x3163d2cfee3183f9874e2869942cc62649eeb004	DCL DCG		ERC721
+\\xe8
```
Comment (aalan3) [inline ethereum/nft/tokens.sql:4808]: Are these emojis gonna work?

**Review by aalan3 — COMMENTED** [review]
Looks good but I see there are a few emojis in here will that work?

File: ethereum/nft/tokens.sql (hunk 1 above)
Comment (masquot) [reply ethereum/nft/tokens.sql:4808]: I understood from @va3093 that unicode and emojis should work.

**Review by aalan3 — APPROVED** [review]

**Review by aalan3 — APPROVED** [review]

**Comment by masquot** [comment]
Token list merged and backfill completed.

**Pull request merged**